### ISSUE TITLE
STT0054: Add speaker profile or metadata to the training data.

### ISSUE THREAD

**Issue by gangagyatso4364** [OpenPecha/stt-catalog-merger#9]
### Description

We need to integrate metadata from the TTS data collection into the training data. This will enhance the creation of a more balanced test set for model evaluation by ensuring the test data is evenly distributed, leading to more accurate model assessments.

We need data from STT: 
    1. [NS](https://docs.google.com/spreadsheets/d/107pF1LcHgbJtCGCdGwffqIyM70eImBmgPOjUuHQfZ7g/edit?gid=769901849#gid=769901849)(Natural speech)
    2. [AB](https://docs.google.com/spreadsheets/d/1yKSzConuVWo8BuMDs2mabF5iiBKUz2wF--LIabFN6QE/edit?gid=0#gid=0)(Audio Book)
    3. [TT](https://docs.google.com/spreadsheets/d/1zX1JEUigHX5gSwfJKqeerhs_5Xxkq8wFbpVUBjrKEnc/edit?gid=0#gid=0) (Tibetan Teaching)
    4. [CS](https://docs.google.com/spreadsheets/d/1-WuaaWKIHJfGQhDXVSwG-v2qGqxQYVqqk7lFeHyxEzk/edit?gid=0#gid=0)(Children Speech)
    5. [NW](https://docs.google.com/spreadsheets/d/1f5KlXPou3UtmRCCEVgwgoJms5WkRiAqb/edit?gid=453963600#gid=453963600)(News)
    6. [PC](https://docs.google.com/spreadsheets/d/1wJ2tPcTec_KnIaN0qcn5fOhBGMzYSHBXmv3kgHYVPlg/edit?gid=0#gid=0)(Podcast)
    7. [MV](https://docs.google.com/spreadsheets/d/1UOhU5Jcge89URmfagDf7Imm_QP74opOx6QsS2LaOSd0/edit?gid=0#gid=0)(Movie)
    8. [HS](https://docs.google.com/spreadsheets/d/1SjlnWsr6YVhBIW6EaS0ITmboDmEuj2VnxGRNL50OPx4/edit?gid=526767285#gid=526767285)(History)

### Completion Criteria

Metadata from the TTS data collection is successfully incorporated into the training data.

### Subtask
- [x] get the current csv file of training data from each department. 
- [x] script to parse the google sheet catalog to get df of audios and it's metadata.
- [x] select list of metadata of audio to be merged. 
- [x] wrtie a script to make the meta values uniform.
- [x] merge the metadata of audio file in catalog with the corresponding audio segment that is derived from the audio. 
- [x] create a documentation of explanation of meta data values. 
- [x] upload to hugging face.
